### PR TITLE
Update Tagalog/Filipino

### PR DESCRIPTION
**Pull request by Ashley434**
There's a grammar.
Neb id: 31412398

### CODE REVIEW

File: res/values-tl/strings.xml [hunk 1]
```diff
@@ -587,7 +587,7 @@
     <string name="edit">"EDIT"</string>
     <string name="start_clan_war">"Simulan ang Clan War"</string>
     <string name="join_clan_war">"Sumali sa Clan War"</string>
-    <string name="none">"Wala"</string>
+    <string name="none">"wala"</string>
```
Comment (Jeii20) [inline res/values-tl/strings.xml:590]: Keep capital letter here

**Review by Jeii20 — CHANGES_REQUESTED** [review]

**Comment by Jeii20** [comment]
Just a council: You don't need to base off on the blue colour texts in every string line (considered the English), so, if it was kept at least as capital letter, keep it capital. If it was kept as small letter, keep it small, and vice-versa. ^^

**Comment by Ashley434** [comment]
Ok,thank you

**Comment by Jeii20** [comment]
PR closed. Another PR with same changes has been identified at #602.

**Pull request closed without merging**